server.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('./webpack.dev.js');
+
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: config.output.publicPath,
+  }),
+);
+
+// Serve the files on port 3000.
+app.listen(3000, () => {
+  /* eslint-disable */
+  console.log('Example app listening on port 3000!\n');
+   /* eslint-enable */
+});
```

Comment:
- [ ] [Optional] Great on doing this, but I think webpack server was sufficient, it was not needed to write your own server. But I like this initiative of going beyond the requirements, just for learning purpose